src/Stratis.Bitcoin.IntegrationTests/ConsensusManagerTests.cs

```diff
@@ -164,8 +165,10 @@ public void ConsensusManager_Fork_Occurs_Node_Gets_Disconnected_Due_To_InvalidSt
                 TestHelper.MineBlocks(minerA, 55);
 
                 // Sync the network to height 55.
-                TestHelper.ConnectAndSync(syncer, minerA);
-                TestHelper.ConnectAndSync(syncer, minerB);
+                TestHelper.Connect(syncer, minerA);
+                TestHelper.Connect(syncer, minerB);
+                TestHelper.WaitLoop(() => TestHelper.AreNodesSynced(syncer, minerA), cancellationToken: new CancellationTokenSource(15 * 60 * 1000).Token);
```

Comment:
How is this different?if cancellation is triggered and not the condition `AreNodesSynced` the test will fail anyway.

Reply:
I think the difference is that we have a bi-directional connection before we attempt to sync. I changed it this way because the other tests do this and seem to be working. I.e. this is consistent with what the other test cases are doing and it works versus failing as it did before.

Reply:
ah I see... I suggest instead to pass an overlaod of the waiting time to the wait method

Reply:
Done.